### ISSUE TITLE
[Request] Handbrake

### ISSUE THREAD

**Issue by NaiveInvestigator** [terrapkg/packages#414]
**What software are you requesting to be packaged?**
https://github.com/HandBrake/HandBrake

**Describe the motivation**
As of now there is no easy way to install handbrake, other than the users compiling it themselves or using flatpak. 

There are some rpm builds on RPMFusion but its not updated nor does it work. Would be awesome if tis add here!

**Comment by madonuko** [comment]
Would you please explain how HandBrake is broken on RPMFusion? What are the things you expected to work but don't?

**Comment by NaiveInvestigator** [comment]
For some videos when i try to encode them  nothing happens when i press the start button, not so sure why. Kinda hit or miss. Also sometimes i don't get any video preview.

 I googled to see if ppl have these issue, to found nothing to fix mine. But I may have missed something tho.

 I also did install `HandBrake-gui` and `HandBrake` too, very confused what to do.

Mainly the newer versions also fixes some NVENC issues, which i do use a lot lol for convenience but rpmfusion doesn't work. 

also would like add that i don't exactly use fedora, but nobara, which is fedora but with some tweaks so that be a cause too?
